### PR TITLE
Update data-classification-content-explorer.md

### PR DESCRIPTION
**Pull request by msbemba**
added portal URL to assign permissions

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3826

File: microsoft-365/compliance/data-classification-content-explorer.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Access to content explorer is highly restricted because it lets you read the con
 > [!IMPORTANT]
 > These permissions supercede permissions that are locally assigned to the items, which allows viewing of the content. 
 
-There are two roles that grant access to content explorer:
+There are two roles that grant access to content explorer and it can be granted using the Microsoft Security and Complaince center(https://protection.office.com/permissions0:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-content-explorer.md]:
```suggestion
There are two roles that grant access to content explorer and it is granted using the Microsoft Security & Compliance Center(https://protection.office.com/permissions):
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/compliance/data-classification-content-explorer.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Access to content explorer is highly restricted because it lets you read the con
 > [!IMPORTANT]
 > These permissions supercede permissions that are locally assigned to the items, which allows viewing of the content. 
 
-There are two roles that grant access to content explorer:
+There are two roles that grant access to content explorer and it is granted using the Microsoft Security & Compliance Center(https://protection.office.com/permissions):
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/data-classification-content-explorer.md]:
```suggestion
There are two roles that grant access to content explorer and it is granted using the [Microsoft Security & Compliance Center](https://protection.office.com/permissions):
```

Comment (JohanFreelancer9) [reply microsoft-365/compliance/data-classification-content-explorer.md]: Apologies @msbemba - my mistake.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Pull request merged**